### ISSUE TITLE
Distorted audio files

### ISSUE THREAD

**Issue by 6Morpheus6** [fishaudio/fish-speech#1002]
### Self Checks

- [x] This template is only for bug reports. For questions, please visit [Discussions](https://github.com/fishaudio/fish-speech/discussions).
- [x] I have thoroughly reviewed the project documentation (installation, training, inference) but couldn't find information to solve my problem. [English](https://speech.fish.audio/) [中文](https://speech.fish.audio/zh/) [日本語](https://speech.fish.audio/ja/) [Portuguese (Brazil)](https://speech.fish.audio/pt/)
- [x] I have searched for existing issues, including closed ones. [Search issues](https://github.com/fishaudio/fish-speech/issues)
- [x] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/fishaudio/fish-speech/issues/515)).
- [x] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [x] Please do not modify this template and fill in all required fields.

### Cloud or Self Hosted

Self Hosted (Source)

### Environment Details

RTX 4070Ti
Windows 10
Python 3.10.16
Pytorch 2.4.1+cu121

### Steps to Reproduce

Generate a couple of times and listen to the audio.

### ✔️ Expected Behavior

Balanced volume. Natural intonation.

### ❌ Actual Behavior

The output audio of the new model is distorted frequently.
This happens with and without Reference Audio files.

This behavior is visible in the wave form as well.

![Image](https://github.com/user-attachments/assets/c1129ee7-3d86-46a9-b862-508e19d54deb)

**Comment by PoTaTo-Mika** [comment]
torch >= 2.5.1 is needed.

**Comment by Whale-Dolphin** [comment]
Can you upload the audio? If it is the model's problem, you can change the issue label to 'not performance well'.

**Comment by 6Morpheus6** [comment]
Thanks a lot for the quick response. :)
@PoTaTo-Mika I switched to pytorch 2.6.0+cu124
@Whale-Dolphin Here is an audio example. It's one out of 3 and the worst one. However, also the other 2 examples had distortions. But this one shows the problem best.
It was generated with default values.
Since wav files aren't allowed to be uploaded on github, I uploaded the audio file to TransferNow and share the link here.
Example Audio: https://www.transfernow.net/dl/20250605bUuJebZA

![Image](https://github.com/user-attachments/assets/a1f933e4-e8a5-444f-846a-f640af9f8f2a)

![Image](https://github.com/user-attachments/assets/275805ca-aabd-488a-82d7-16cf3ee0f002)

PS: @Whale-Dolphin I found no way to change the label. Maybe you can do it on your side.

**Comment by Stardust-minus** [comment]
We fixed an existed problem in #1014 , and you can retry to generate. Applogize again for our mistake.
